File: internal/content/testdata/bench/typescript/is-armstrong-3digit.ts
function is_armstrong_3digit(n: number): boolean {
    if (n < 100 || n > 999) return false;
    let s = 0;
    let temp = n;
    while (temp > 0) {
        const r = temp % 10;
        s += r * r * r;
        temp = Math.floor(temp / 10);
    }
    return s === n;
}

File: internal/content/testdata/bench/go/is-armstrong-3digit.go
func is_armstrong_3digit(n int) bool {
	if n < 100 || n > 999 {
		return false
	}
	s := 0
	temp := n
	for temp > 0 {
		r := temp % 10
		s += r * r * r
		temp /= 10
	}
	return s == n
}
